vulnerable_people_form/integrations/postcode_eligibility.py

```diff
@@ -1,6 +1,15 @@
 import os
 
+from .persistence import get_rds_data_client, generate_string_parameter, execute_sql
+
 
 def check_postcode(postcode):
-    # TODO: add implementation
-    return os.environ.get("FAIL_POSTCODE_VERIFICATION") is None
+    records = execute_sql(
+        "CALL is_postcode_on_lockdown(:postcode)",
```

Comment:
i think this should be  "CALL cv_base.is_postcode_on_lockdown(:postcode)"

Reply:
It appears that it has been silently renamed in the relevant `covid-engineering` PR, although not in the PR description. As the PR is not yet merged, this was not known before, as we only had the "it will be like this" info given previously. Updated! 🙂